course_discovery/apps/api/v1/tests/test_views/test_course_runs.py

```diff
@@ -135,6 +141,24 @@ def test_list(self):
             self.serialize_course_run(CourseRun.objects.all().order_by(Lower('key')), many=True)
         )
 
+    def test_list_edx_org_short_name_filter(self):
+        """
+        Verify the endpoint returns a filtered list of course runs based on edx organization.
```

Comment:
Which endpoint?
Ideally docsrings should have max length of 80 characters, update doscstring.

https://edx.readthedocs.io/projects/edx-developer-guide/en/latest/style_guides/python-guidelines.html#docstrings

Reply:
Done